### PR TITLE
[ashell] Fix for calling eval after starting a new application

### PR DESCRIPTION
**Pull request by brianjjones**
Currently jerry_eval will fail when ashell runs it after cleaning
the jerry engine.  This fixes it.

Signed-off-by: Brian J Jones <brian.j.jones@intel.com>

### CODE REVIEW

File: src/ashell/jerry-code.c [hunk 1]
```diff
@@ -326,8 +326,17 @@ void restore_zjs_api() {
 #ifdef BUILD_MODULE_SENSOR
     zjs_sensor_init();
 #endif
+    jerry_value_t global_obj = jerry_get_global_object();
```
Comment (jimmy-huang) [inline src/ashell/jerry-code.c]: should just use ZVAL here, so you don't have to free it, this is a weird place to set this also, shouldnt this be in the zjs_modules_init() function?

Comment (brianjjones) [inline src/ashell/jerry-code.c]: This is what is done in main.c right now.  I'm fine with moving it to modules_init, but that's for another patch.

Comment (brianjjones) [inline src/ashell/jerry-code.c]: @jimmy-huang https://github.com/01org/zephyr.js/pull/1044 will take care of this.

**Comment by brianjjones** [comment]
https://github.com/01org/zephyr.js/pull/1044 takes this fix and adds it to the moving of the init code.

**Pull request closed without merging**